tests/runtest.cmd

```diff
@@ -464,6 +465,7 @@ echo                               2: GC on transitions to preemptive GC
 echo                               4: GC on every allowable JITed instruction
 echo                               8: GC on every allowable NGEN instruction
 echo                              16: GC only on a unique stack trace
+echo tieredcompilation         - Run the tests with COMPlus_EXPERIMENTAL_TieredCompilation=1
```

Comment:
I think the alignment is correct, it should be the same column as gcsimulator's alignment. gcstresslevel is different because it has multi-char escape characters